Restore IE 6/7/8 compatibility

wordofchristian opened the pull request:
Fixes #104

wordofchristian commented:
I signed it!

maksimr commented:
@wordofchristian  thanks a lot!
Fix eslint errors please - https://travis-ci.org/karma-runner/karma-mocha/jobs/149877795#L611

dignifiedquire commented:
Thank you, it seems the tests are currently failing, also could you please make sure that your commits follow our [convention](http://karma-runner.github.io/1.0/dev/git-commit-msg.html).

wordofchristian commented:
I updated the commits (forgive the force push), and fixed the test failure that I introduced. The remaining test failures are already present on master and I'm not sure how the fix them yet.

maksimr commented:
LGTM

dignifiedquire commented:
I just merged #106, could you please rebase onto `master` which should hopefully fix travis

wordofchristian commented:
done

dignifiedquire commented:
Thanks :octocat:
